### PR TITLE
add robot-compile-setup.sh

### PR DESCRIPTION
**Pull request by k-okada**

### CODE REVIEW

**Comment by 130s** [comment]
I'm afraid the typo in file name :dizzy_face:

**Comment by k-okada** [comment]
sorry , fixed commit log

File: hironx_ros_bridge/robot/robot-compile-setup.sh [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+mv ${TMPDIR}/rtmros_nextage-0.2.14/nextage_description/ ${TMPDIR}/opt/jsk/etc/
+tar -C ${TMPDIR} -cvzf ${TMPDIR}/opt-jsk-base-model.tgz ./opt/jsk/
+
+commands="
+  . ~/.profile;
+  env;
+  trap 'exit 1' ERR;
+  set +x;
+  echo \"* setup /opt/jsk directory *\";
+  su -c 'mkdir -p /opt/jsk; chown $userid /opt/jsk';
```
Comment (130s) [inline hironx_ros_bridge/robot/robot-compile-setup.sh]: Sorry not to have mentioned earlier; `/opt/jsk` folder is made by manufacturer (they then assign our user account to the folder that can be used only for installing opensource controller). So I think we can remove this line (and the echo line right above).

Comment (k-okada) [inline hironx_ros_bridge/robot/robot-compile-setup.sh]: ok, updated

File: hironx_ros_bridge/robot/robot-compile-setup.sh [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+userid=$2
+userid=${userid:="hiro"} 
+
+DATE=`date +%Y-%m-%d`
+
+TMPDIR=/tmp/HOGE
+mkdir -p ${TMPDIR}/opt/jsk
+#wget https://github.com/tork-a/rtmros_nextage/archive/0.2.14.zip -O ${TMPDIR}/rtmros_nextage-0.2.14.zip || echo "ERROR:: Failed to download source code"
+#wget 'https://docs.google.com/uc?authuser=0&id=0B5hXrFUpyR2iZS0tQlFyXzhjaGc&export=download' -O ${TMPDIR}/opt-jsk-base.tgz || echo "ERROR:: Failed to download source code"
+
```
Comment (k-okada) [inline hironx_ros_bridge/robot/robot-compile-setup.sh:36]: uncomment

**Comment by 130s** [comment]
Very useful! Files got installed are:

```
$ du /opt/jsk/ 
            3 /opt/jsk/etc/nextage_description/www
         5091 /opt/jsk/etc/nextage_description/urdf/meshes
         5128 /opt/jsk/etc/nextage_description/urdf
         5522 /opt/jsk/etc/nextage_description/models
        10671 /opt/jsk/etc/nextage_description
        10673 /opt/jsk/etc
          218 /opt/jsk/lib
            9 /opt/jsk/include/uuid
           12 /opt/jsk/include/eigen3/unsupported/Eigen/src/NumericalDiff
           10 /opt/jsk/include/eigen3/unsupported/Eigen/src/MoreVectorization
           48 /opt/jsk/include/eigen3/unsupported/Eigen/src/FFT
           74 /opt/jsk/include/eigen3/unsupported/Eigen/src/AutoDiff
           28 /opt/jsk/include/eigen3/unsupported/Eigen/src/IterativeSolvers
           83 /opt/jsk/include/eigen3/unsupported/Eigen/src/MatrixFunctions
          283 /opt/jsk/include/eigen3/unsupported/Eigen/src/SparseExtra
           49 /opt/jsk/include/eigen3/unsupported/Eigen/src/BVH
          145 /opt/jsk/include/eigen3/unsupported/Eigen/src/NonLinearOptimization
           59 /opt/jsk/include/eigen3/unsupported/Eigen/src/Polynomials
          154 /opt/jsk/include/eigen3/unsupported/Eigen/src/Skyline
          947 /opt/jsk/include/eigen3/unsupported/Eigen/src
         1130 /opt/jsk/include/eigen3/unsupported/Eigen
         1132 /opt/jsk/include/eigen3/unsupported
           32 /opt/jsk/include/eigen3/Eigen/src/Jacobi
          459 /opt/jsk/include/eigen3/Eigen/src/Sparse
           78 /opt/jsk/include/eigen3/Eigen/src/SVD
          216 /opt/jsk/include/eigen3/Eigen/src/Eigen2Support/Geometry
          386 /opt/jsk/include/eigen3/Eigen/src/Eigen2Support
          110 /opt/jsk/include/eigen3/Eigen/src/QR
          271 /opt/jsk/include/eigen3/Eigen/src/Core/util
          408 /opt/jsk/include/eigen3/Eigen/src/Core/products
          117 /opt/jsk/include/eigen3/Eigen/src/Core/arch/SSE
            7 /opt/jsk/include/eigen3/Eigen/src/Core/arch/Default
           58 /opt/jsk/include/eigen3/Eigen/src/Core/arch/AltiVec
           52 /opt/jsk/include/eigen3/Eigen/src/Core/arch/NEON
          236 /opt/jsk/include/eigen3/Eigen/src/Core/arch
         2354 /opt/jsk/include/eigen3/Eigen/src/Core
          101 /opt/jsk/include/eigen3/Eigen/src/plugins
           56 /opt/jsk/include/eigen3/Eigen/src/Householder
          292 /opt/jsk/include/eigen3/Eigen/src/Eigenvalues
           24 /opt/jsk/include/eigen3/Eigen/src/misc
           45 /opt/jsk/include/eigen3/Eigen/src/StlSupport
           29 /opt/jsk/include/eigen3/Eigen/src/LU/arch
          162 /opt/jsk/include/eigen3/Eigen/src/LU
           61 /opt/jsk/include/eigen3/Eigen/src/Cholesky
           11 /opt/jsk/include/eigen3/Eigen/src/Geometry/arch
          376 /opt/jsk/include/eigen3/Eigen/src/Geometry
         4538 /opt/jsk/include/eigen3/Eigen/src
         4609 /opt/jsk/include/eigen3/Eigen
         5744 /opt/jsk/include/eigen3
         5755 /opt/jsk/include
        16648 /opt/jsk
```

**Pull request merged**